pkg/test/settings.go

```diff
@@ -51,7 +51,7 @@ func Settings(overrides ...SettingOptions) *azsettings.Settings {
 		ClusterID:                      lo.FromPtrOr(options.ClusterID, "00000000"),
 		KubeletClientTLSBootstrapToken: lo.FromPtrOr(options.KubeletClientTLSBootstrapToken, "test-token"),
 		SSHPublicKey:                   lo.FromPtrOr(options.SSHPublicKey, "test-ssh-public-key"),
-		NetworkPlugin:                  lo.FromPtrOr(options.NetworkPlugin, "kubenet"),
+		NetworkPlugin:                  lo.FromPtrOr(options.NetworkPlugin, "azure"),
```

Comment:
this change means that you will need to change all the max-pods flags in your unit tests to "250". Currently there is a bug that will set them to 110(Default) https://github.com/Azure/karpenter-provider-azure/blob/main/pkg/providers/imagefamily/resolver.go#L37 change this value to "azure" and the max pods to 250 and you should be good